Example/ExampleApp/Questionnaire/ORKOrderedTaskView.swift

```diff
@@ -24,13 +25,44 @@ struct ORKOrderedTaskView: UIViewControllerRepresentable {
     init(
         tasks: ORKOrderedTask,
         delegate: ORKTaskViewControllerDelegate,
-        tintColor: Color = Color(UIColor(named: "AccentColor") ?? .tintColor)
+        tintColor: Color = Color(UIColor(named: "AccentColor") ?? .tintColor),
+        outputDirectory: URL? = nil
     ) {
         self.tasks = tasks
         self.tintColor = tintColor
         self.delegate = delegate
+        self.outputDirectory = outputDirectory
+    }
+
+    func getTaskOutputDirectory(_ taskViewController: ORKTaskViewController) -> URL? {
+        do {
+            let defaultFileManager = FileManager.default
+
+            // Identify the documents directory.
+            let documentsDirectory = try defaultFileManager.url(
+                for: .documentDirectory,
```

Comment:
I would suggest that we use the `.cachesDirectory` for storing the images. The documents directly might be synced or actively used by an application that uses the framework to store user information which might conflict with your use case.

Theoretically, the `.itemReplacementDirectory` parameter would be the best choice here as we should just hold on to the files for a very short time but using the `.cachesDirectory` would also be a reasonable choice with a bit longer guarantee that files would be persisted a bit longer. Depends on how we decide on my comment below.

We should maybe add an extension method to `ORKTaskViewController` to delete the files in the temporary directory. We can add to the documentation that this should be called within the `ORKTaskViewControllerDelegate` subclass's `taskViewController(_ taskViewController: ORKTaskViewController, didFinishWith reason: ORKTaskViewControllerFinishReason, error: Error?)` method after extracting the result and make it the responsibility of the caller to move the files to a directory where they can be persisted for a longer time. We don't want to be responsible for sensitive user data lying somewhere on disk as part of this framework and should put that responsibility to the user of the framework and therefore clean out the directory after we have passed the files on to the caller. If that would be the approach, I would suggest using `.itemReplacementDirectory`.

What do you think about this approach?

Reply:
This sounds like a reasonable approach to me. In the case that we use a temporary directory, the caller would not only be responsible for moving the files to another directory for long-term storage, but also to update the `QuestionnaireResponse` object with the new file URL.

Reply:
Exactly, that is an other very good point! 🚀👍